srcs/incs/HttpRequest.hpp

```diff
@@ -0,0 +1,50 @@
+#ifndef SRCS_INCS_HTTPREQUEST_HPP_
+#define SRCS_INCS_HTTPREQUEST_HPP_
```

Comment:
The linter was throwing this error:
```
srcs/incs/HttpRequest.hpp:1:  #ifndef header guard has wrong style, please use: SRCS_INCS_HTTPREQUEST_HPP_  [build/header_guard] [5]
srcs/incs/HttpRequest.hpp:50:  #endif line should be "#endif  // SRCS_INCS_HTTPREQUEST_HPP_"  [build/header_guard] [5]
```

We could add the `-build/header_guard` filter to the linter config or use `#pragma once` instead of header guards.

Reply:
what does pragma once do?